### PR TITLE
Add product_category_product_link

### PR DESCRIPTION
**Pull request by nuprodRomainF**
new file:   product_category_product_link/__init__.py
	new file:   product_category_product_link/__manifest__.py
	new file:   product_category_product_link/i18n/ca.po
	new file:   product_category_product_link/i18n/product_category_product_link.pot
	new file:   product_category_product_link/models/__init__.py
	new file:   product_category_product_link/models/product_category.py
	new file:   product_category_product_link/models/product_template.py
	new file:   product_category_product_link/readme/CONTRIBUTORS.rst
	new file:   product_category_product_link/readme/DESCRIPTION.rst
	new file:   product_category_product_link/static/description/icon.png
	new file:   product_category_product_link/static/description/index.html

### CODE REVIEW

**Comment by nuprodRomainF** [comment]
Add setup

**Comment by FrancoMaxime** [comment]
hi @nuprodRomainF,
Thanks for contributing. That's not the best way to migrate a module from a previous version. By copying the files, we lose a lot of informations (previous commit, commiters, all the history). 

I think you should take a look at this link [https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0](url)
If you  have any questions, i will answer it

**Review by FrancoMaxime — CHANGES_REQUESTED** [review]
related to comment: https://github.com/OCA/product-attribute/pull/1420#issuecomment-1735030853

**Pull request closed without merging**
